app/models/team.rb

```diff
@@ -3,4 +3,12 @@ class Team < ActiveRecord::Base
   has_many :income
   has_many :bonus_credit
   has_many :public_relation
+
+  def self.countries
```

Comment:
Probably have to change the method name to like all_team_names or something in the future.